src/Tests/RegistrationTest.php

```diff
@@ -46,6 +46,7 @@ protected function setUp() {
 
     $this->user_config = $this->config('user.settings');
     $this->acquisition_config = $this->config('decoupled_auth.settings');
+    $this->acquisition_config->set('unique_emails.mode', 'none')->save();
```

Comment:
Think we need to think more about the tests as acquisitions should work with all the various modes...